lib/manager/npm/extract/index.js

```diff
@@ -131,7 +131,8 @@ async function extractPackageFile(content, fileName, config) {
   if (lernaJson) {
     lernaDir = path.dirname(fileName);
     lernaPackages = lernaJson.packages;
-    lernaClient = lernaJson.npmClient || lockFiles.yarnLock ? 'yarn' : 'npm';
+    lernaClient =
```

Comment:
> npmClient: an option to specify a specific client to run commands with (this can also be specified on a per command basis). Change to "yarn" to run all commands with yarn. Defaults to "npm".
> https://github.com/lerna/lerna#lernajson